### PR TITLE
Add pages & wikilinks info

### PR DESCRIPTION
**Pull request by Ewoutvans**
Draft for https://github.com/SpongePowered/Ore/pull/443

### CODE REVIEW

File: source/ore/publish.rst [hunk 1]
```diff
@@ -145,3 +145,57 @@ Support
...
+Needs Changes
+-------------
+
+Ore staff can hide your project and request changes the changes in question will be listed in the banner. This state will hide your project from the 
+public if you are done you can send your project for approval. 
+
+Needs Approval
+--------------
+
+This state will hide your project from the public while a staff member is going to validate you made the changes that where requested.
```
Comment (ST-DDT) [inline source/ore/publish.rst]: I would rephrase that a little bit.
`In this state your project is still hidden from the public while a staff member is going to validate you made the changes that where requested. Please be patient, the staff members might be busy with other tasks. If the project gets approved it will automatically become visible again. Please make sure to actually fix the points that require changes, otherwise <What happens otherwise?>`

File: source/ore/publish.rst [hunk 2]
```diff
@@ -145,3 +145,57 @@ Support
 -------
 
 Support cannot do anything, merely a way of showing that the member is a part of the project.
+
+
+Documenting Your Project With Pages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To document your plugin Ore offers the creation of pages, by default you get a 'Home' page when you create your new Project.
```
Comment (Inscrutable) [inline source/ore/publish.rst]: You seem to use a comma where a full stop (or semicolon) belongs. Two separate sentences make more sense.
"To document your plugin Ore offers the creation of pages. By default you get a 'Home' page when you create your new Project." 
This happens in a few places ...

File: source/ore/publish.rst [hunk 3]
```diff
@@ -145,3 +145,57 @@ Support
...
+To add new pages you can click on the '+' icon in the Pages bar on the right of the screen, doing so will open a pop-up where you
```
Comment (Inscrutable) [inline source/ore/publish.rst]: Again, please replace the comma. A semicolon would do here "... of the screen; doing so will ..."

File: source/ore/publish.rst [hunk 4]
```diff
@@ -145,3 +145,57 @@ Support
...
 Support cannot do anything, merely a way of showing that the member is a part of the project.
+
+
+Documenting Your Project With Pages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To document your plugin Ore offers the creation of pages, by default you get a 'Home' page when you create your new Project.
+To add new pages you can click on the '+' icon in the Pages bar on the right of the screen, doing so will open a pop-up where you 
+can choose a name and the parent page (adding a page on the Home page is not possible). If you choose the '<None>' option the page 
+will become a root page where you can later add child pages on, if you hover choose to add it to an other page the page will only show
```
Comment (Inscrutable) [inline source/ore/publish.rst]: Again, please replace comma with full stop and begin a new sentence. "If you hover  ..."
Secondly, I'm not sure exactly what "If you hover choose to add it" means ...  can you rephrase that so it's clear?
Also, please replace "an other" -> "another"
Add comma, lose one "page"  - i.e. "... to another page, it will only appear when the parent page is expanded"

File: source/ore/publish.rst [hunk 5]
```diff
@@ -145,3 +145,57 @@ Support
...
+when the parent page is expanded.
+To document you pages you can use CommonMark.
```
Comment (Inscrutable) [inline source/ore/publish.rst]: "To document *your* pages" ...

File: source/ore/publish.rst [hunk 6]
```diff
@@ -145,3 +145,57 @@ Support
...
+
+Linking pages
+-------------
+
+If you have the following tree structure in you pages:
```
Comment (Inscrutable) [inline source/ore/publish.rst]: *your*
I suggest changing this a bit so the subsequent sentence does not start with "And".
eg. "Let's assume you have the following tree structure in your pages:"
Then we can change the sentence starting with "And" into "Now, if you ..."

File: source/ore/publish.rst [hunk 7]
```diff
@@ -145,3 +145,57 @@ Support
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To document your plugin Ore offers the creation of pages, by default you get a 'Home' page when you create your new Project.
+To add new pages you can click on the '+' icon in the Pages bar on the right of the screen, doing so will open a pop-up where you 
+can choose a name and the parent page (adding a page on the Home page is not possible). If you choose the '<None>' option the page 
+will become a root page where you can later add child pages on, if you hover choose to add it to an other page the page will only show
+when the parent page is expanded.
+To document you pages you can use CommonMark.
+
+Linking pages
```
Comment (Inscrutable) [inline source/ore/publish.rst]: Use Sentence Case For Titles, Please.

File: source/ore/publish.rst [hunk 8]
```diff
@@ -145,3 +145,57 @@ Support
...
+link that is guaranteed to work.
+
+.. note::
+    You can also specify a title for the link and separate it with a Pipe symbol so `[[Title|Link]]` will become a link with the title
+
+.. note::
+    WikiLinks are only supported in Ore so if you use them on the Home-page they will not be displayed correctly on the forums.
+    However, we still recommend using WikiLinks instead of normal links to guarantee the future working of your wiki.
+
+Project states
```
Comment (Inscrutable) [inline source/ore/publish.rst]: Sentence Case, Please

File: source/ore/publish.rst [hunk 9]
```diff
@@ -145,3 +145,57 @@ Support
...
+~~~~~~~~~~~~~~~
+
+When creating a new project a banner will appear telling you your project is in a 'new' state.
+The new state is intended to indicated to users and admins you are still working on the setup of your project. If you are done documenting
```
Comment (Inscrutable) [inline source/ore/publish.rst]: "If you have finished documenting" ...
Also, please add a comma after "comply with the guidelines"
I think saying "click the 'publish' button to complete the process" may be an improvement, too.

File: source/ore/publish.rst [hunk 10]
```diff
@@ -145,3 +145,57 @@ Support
...
+    WikiLinks are only supported in Ore so if you use them on the Home-page they will not be displayed correctly on the forums.
+    However, we still recommend using WikiLinks instead of normal links to guarantee the future working of your wiki.
+
+Project states
+~~~~~~~~~~~~~~~
+
+When creating a new project a banner will appear telling you your project is in a 'new' state.
+The new state is intended to indicated to users and admins you are still working on the setup of your project. If you are done documenting 
+your plugin to make it comply with the guidelines you can click the Button 'Publish' to mark you are done.
+If you however don't Publish it the project will be published after 24 hours since you created the project.
```
Comment (Inscrutable) [inline source/ore/publish.rst]: "However, if you don't publish it yourself then the project will be published automatically 24 hours after it's creation."

File: source/ore/publish.rst [hunk 11]
```diff
@@ -145,3 +145,57 @@ Support
...
+
+Needs Changes
+-------------
+
+Ore staff can hide your project and request changes the changes in question will be listed in the banner. This state will hide your project from the
```
Comment (Inscrutable) [inline source/ore/publish.rst]: Semicolons needed between "... changes; the changes ..."  and also
"This state will hide your project from the public;  if you are done ..."

File: source/ore/publish.rst (hunk 1 above)
Comment (Inscrutable) [inline source/ore/publish.rst]: The case is all messed up here.
It should be "while waiting for a staff member to review and validate the changes that were requested".

**Review by Inscrutable — CHANGES_REQUESTED** [review]
Commas and semicolons will fix most of the run-on sentences. Please make sure Titles are in Title Case, folks.

**Review by ST-DDT — APPROVED** [review]

**Review by Inscrutable — APPROVED** [review]
